helm/metac/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+name: metac
+apiVersion: v1
```

Comment:
I guess, we had some issues w.r.t helm version mapping to corresponding app version.
How does this apiVersion translate to while running the helm install command?
Does this mean, users will specify v1 while installing metac via helm?

Reply:
This comes with the `helm create` command that generates the chart scaffolding. I believe it refers to the tiller api version.